### PR TITLE
Bump pip from 20.3.2 to 20.3.3

### PR DESCRIPTION
**Pull request by ybiquitous**
- https://github.com/pypa/pip/blob/20.3.3/NEWS.rst
- https://github.com/pypa/pip/compare/20.3.2...20.3.3

NOTE: Dependabot cannot bump `pip` for any reason. 🤔 

> No update possible for pip 20.3.2

-- https://github.com/sider/devon_rex/network/updates/80499782

### CODE REVIEW

**Comment by ybiquitous** [comment]
@yykamei Do you know the reason that Dependabot cannot bump it?

**Comment by yykamei** [comment]
Sorry, I said "it works," but it's incorrect. According to Pipenv, `pip` is listed in `BAD_PACKAGES`, which are excluded from installation lists.

https://github.com/pypa/pipenv/blob/deefb637a646b4e5432e1934453da61a9683ad7e/pipenv/core.py#L51-L58

From this result, we cannot manage Pip through Pipenv :disappointed:

File: python/Pipfile [hunk 1]
```diff
@@ -4,5 +4,5 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [packages]
-pip = "==20.3.2"
+pip = "==20.3.3"
```
Comment (ybiquitous) [inline python/Pipfile:7]:
```suggestion
# NOTE: pip cannot be installed via pipenv.
#       See <https://github.com/pypa/pipenv/blob/v2020.11.15/pipenv/core.py#L51-L58>
pip = "==20.3.3"
```

**Comment by ybiquitous** [comment]
@yykamei Thanks, I got it. 👍 

Which will we easily manage the dependencies? As the following suggestion, does it make sense to manage `pip` in Pipfile still?
https://github.com/sider/devon_rex/pull/363/files#r551718527

**Comment by yykamei** [comment]
:thinking: Using `requirements.txt` might be a better approach because Dependabot seems to support it. We can just install Pip and Pipenv like this:

```console
pip install -r requirements.txt
```

**Comment by ybiquitous** [comment]
> because Dependabot seems to support it

Where is it documented? I cannot find it in <https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates>...

**Comment by ybiquitous** [comment]
Ah, I find an example for `requirements.txt` update:
https://github.com/justdjango/dream_blog/commit/4b7d1ddbf99df5e794f86f5313321b31877ddcb7

Sounds good. I will try it. 💪

**Pull request closed without merging**